packages/canari/PKGBUILD

```diff
@@ -16,6 +16,7 @@ sha1sums=('1ace7a39055b5245e95f520d75dab01d8181b50b')
 
 build(){
   cd "$srcdir/canari-$pkgver"
+  pip2 install pyasn1 # temporal workaround to avoid setuptool issues
```

Comment:
remove this line: `pip2 install pyasn1 # temporal workaround to avoid setuptool issues` and add to `depends` the package `python2-pyasn1`. That would be a better approach than installing from `pip` since we have that package on the official repository.